packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -236,3 +236,104 @@ def get_task_url(dcos_api_session, task_name, stream=False):
     return '/system/v1/agent/{}/logs/v1/{}/framework/{}/executor/{}/container/{}'.format(slave_id, endpoint_type,
                                                                                          framework_id, executor_id,
                                                                                          container_id)
+
+
+def test_log_v2_text(dcos_api_session):
```

Comment:
Don't integration tests require type annotations now?

Reply:
Talked with @mnaboka and I think if the linters pass, this can be ignored. it's possible that the type annotations are only enforced on the EE integration tests.

Reply:
Indeed there is not a mypy test configured for this repo, so this only applies in EE